tests/baselines/reference/variadicTuples1.types

```diff
@@ -400,7 +400,7 @@ function f3<T extends unknown[]>(t: [string, ...T, number]) {
 
     let [c1, c2, ...cx] = t;  // string, [string, ...T, number][1], (number | T[number])[]

```

Comment:
This comment is now wrong (and I think others in this PR too); may want to update them.

Reply:
good catch, I fixed the ones corresponding to the baselines changes